data/hai/hai reveal 4 middle.txt

```diff
@@ -1114,46 +1140,190 @@ mission "Hai Reveal [A09] History of Appeasement"
+				goto questions
+
+			# Third phase of the conversation: decide to fix the problem.
+			label fix
+			action
+				set "Hai Reveal [A09] History of Appeasement: label fix"
+			`	Sayari explains. "The grievance is against the Elders. There's a process to seek redress of grievances with the Elders. Teeneep needs to at least attempt to follow that process, and if she does not go through with normal resolution channels, she will be censured from public life. She must confront the Elders directly."`
+			`	Teeneep makes the effort to give a final, defiant snarl of disgust. "She's right," she grates out. "I hate it, but she's right. Especially because several of those reports indicate that the Navy knew of Hai on some ships and stood by while they were taken away. Which makes it your business, too."`
+			`	Thinking back to your meeting with Katya, you realize this is probably a step closer to a "hostile diplomatic incident."`
+				goto questions
+			label media
+			action
+				set "Hai Reveal [A09] History of Appeasement: label media"
+			`	Sayari gives you a quizzical look. "Of course they do, though Tee has access to more efficient options. That's not the point, though."`
+				goto questions
+			label go
+			`	Sayari nods at you and looks toward Teeneep. "Look, the Elders are pretty much permanently on call at the moment. None of them are leaving <origin>, and they're in session as we speak. Let me put a priority audience request into the system, and you can probably see them this afternoon. In the meantime, why don't I order some food here? You can use my spare desk over there to prepare what you need to say."`
+			`	Teeneep takes a deep breath and accepts the plan with sudden and surprising grace. She gathers her pads, then sets up where indicated. "Alright, just get me some comfort food while I work on this."`
+			`	Sayari asks you, "Her favorite foods are acorn curry and nutmeg pie. Which should we order?"`
+			choice
+				`	"Acorn curry sounds... good?"`
+					goto curry
+				`	"I'd love some nutmeg pie."`
+			`	The pie tastes and feels like a meaty shepherd's pie, even though it's all nuts and vegetables. The subtle taste of nutmeg really brings it all together, accenting the acorns quite well.`
+				goto meeting
+			label curry
+			`	You eat the most delicious curry you've ever tasted, with a perfect combination of acorns and the best Indian spices. It's amazing what Hai can do with nuts; no wonder Teeneep considers this a comfort food.`
+
+			# Scene change: the meeting with the Elders begins.
+			label meeting
+			`	After a short hour of frantic work, Teeneep leads you to a room within the government complex. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."` 
```

Comment:
Earlier on we imply that the elder council is somewhat large, here we have an extremely small selection. I would suggest making this either 5 or 7, though with a larger number perhaps 3 have some reserve power.

Reply:
This is not the Elder Council. It is only a few Elders the greater council chose to make a quick decision on a matter. The larger the group, the slower they'll take to get to a consensus. Three is the smallest group size that won't result in a tie.

I'm not clear on if you think this is the entire Elder Council, or if you're just asking for a larger group.

Reply:
I have added a dialog option for Sayari to expound on the Special Adjunctive Triad. It also gives the lore justification for Teeneep's "special Elder powers" when she was out in the field.

"Our governmental structure is directed towards stability, but sometimes there's a problem that merits rapid action. When needed, the Council will nominate a group of Elders, or even just one, to take actions the full Council would otherwise spend weeks deliberating upon. Their power is limited, and is defined when they're nominated, but their dictate is immediate. If necessity pushes beyond their preset limits, they can make a recommendation with a fast track to a full Council discussion. In trying times like these, they may get a meeting by tomorrow."

Reply:
My thinking was that 3 is an extremely small number for a serious deliberation, but with the additional justification I suppose that does work somewhat neatly.